ChangeLog.md

```diff
@@ -20,6 +20,12 @@ See docs/process.md for more on how version tagging works.
 
 3.1.54 (in development)
 -----------------------
+- The type of `EMSCRIPTEN_WEBGL_CONTEXT_HANDLE` was changed to unsigned and
+  the only valid error returned from `emscripten_webgl_create_context` is
+  now zero.  This allows `EMSCRIPTEN_WEBGL_CONTEXT_HANDLE` to hold a pointer
+  to memory even in 2GB+ mode.  Since `emscripten_webgl_create_context` never
+  returns anything except zero for its errors today this change should not
+  require any action. (#21268)
```

Comment:
Hmm interesting.. Originally the intent was that it would have been returning the same set of negative return codes, i.e. https://github.com/emscripten-core/emscripten/blob/21f25339df9c6aa8c288297de73236e6aaa79582/system/include/emscripten/em_types.h#L41-L48 to allow differentiating between "WebGL is not supported" and "you just inputted a selector to a DOM element that doesn't exist", but looks like indeed that distinction doesn't exist (or maybe has been lost in a refactor)

LGTM overall, looks like debug builds have prints that mention which kind of failure it was, so I think we'll go with those.